package.json

```diff
@@ -10,5 +12,8 @@
     "co": "4.5.x",
     "data.task": "3.0.x",
     "ramda": "0.15.x"
+  },
+  "devDependencies": {
+    "babel": "^5.8.9"
```

Comment:
I don't think that's a good idea. It allows npm to download versions less than`5.8.9`